### PR TITLE
[14.0][MIG] stock_location_children

### PR DESCRIPTION
**Pull request by hparfr**

### CODE REVIEW

**Comment by hparfr** [comment]
ping @grindtildeath @guewen

**Review by rousseldenis — APPROVED** [review]

**Review by bguillot — APPROVED** [review]

**Review by bealdav — APPROVED** [review]
LGTM

**Review by HaraldPanten — APPROVED** [review]

File: stock_location_children/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Stock location children",
+    "summary": "Add relation between stock location and all its children",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
```
Comment (LoisRForgeFlow) [inline stock_location_children/__manifest__.py]: Still Alpha?

**Comment by hparfr** [comment]
@LoisRForgeFlow status changed as requested

**Review by LoisRForgeFlow — APPROVED** [review]
LG!

**Pull request merged**